### DISCUSSION TITLE
Color of the selection in the explorer

### DISCUSSION THREAD

**Discussion by rumugius** [Maplespe/DWMBlurGlass#409] [Q&A]
Is there any way to make the selection in the explorer be non-solid color?
So, now it looks like this:
![image](https://github.com/user-attachments/assets/cb7a016c-ebbd-4fd4-a9ce-18bc74aca202)

But it would be really cool if it looked something like this:
![Без имени-1](https://github.com/user-attachments/assets/e2afebfb-f01b-4467-b367-7f2782b2aa5d)

**Comment by ALTaleX531** [comment]
This has to do with your msstyle, not with DWMBlurGlass itself.

**Comment by Undisputed00x** [comment]
`ListItem` subclass is what you want to edit 
<details>
<summary>

### Msstyles Editor Picture
</summary>
<p>

![Screenshot 2024-11-10 133349](https://github.com/user-attachments/assets/14a0cd5f-0580-4501-b6e7-45f4ee9f8763)

</p>
</details>
